segmentation/utils/spatial_analysis_test.py

```diff
@@ -33,13 +34,10 @@ def make_distance_matrix(enrichment_type):
         # are located close in proximity to 10 cells mostly positive for marker 2.
         # Other included cells are not significantly positive for either marker and are located
         # far from the two positive populations.
-        dist_mat_pos = np.zeros((80, 80))
-        dist_mat_pos[10:20, :10] = 50
-        dist_mat_pos[:10, 10:20] = 50
-        dist_mat_pos[20:40, :20] = 200
-        dist_mat_pos[:20, 20:40] = 200
-        dist_mat_pos[40:80, :40] = 300
-        dist_mat_pos[:40, 40:80] = 300
+
+        dist_mat_pos = synthetic_spatial_datagen.direct_init_dist_matrix(num_A=10, num_B=10, num_C=60,
+                                                                         distr_AB=(int(dist_lim / 5), 1),
+                                                                         distr_random=(int(dist_lim * 5), 1))
```

Comment:
Idk if the test functions are randomly failing on asserts about `p_pos`/`p_neg`/z-scores, but if they are, then increasing the amount of `num_X` should decrease how often those tests fails.  Testing on 10 As and 10 Bs seems a little small, i.e susceptible to significant sampling variations.

This goes for the other cases in `make_distance_matrix` as well

Reply:
We can definitely up the ante in the future. I've added an issue to address the current hard-coding of expression matrices (currently assuming 80 entries per FOV in positive enrichment and 60 cells per FOV in negative enrichment). After we come up with a scheme to synthetically generate these as well, it might be easier to generate variable-length testing dist and expression matrices and thus test on more data.